docs-web/app-protect/troubleshooting.md

```diff
@@ -0,0 +1,95 @@
+```
+Note that in the events section, we have a `Normal` event with the `AddedOrUpdated` reason, which informs us that the configuration was successfully applied.
+
+### Checking events of APPolicy
+
+After you create or update an APPolicy, you can immediately check if the NGINX configuration was successfully applied by NGINX:
+```
+$ kubectl describe appolicy dataguard-alarm
+Name:         dataguard-alarm
+Namespace:    default
+. . . 
+Events:
+  Type    Reason          Age    From                      Message
+  ----    ------          ----   ----                      -------
+  Normal  AddedOrUpdated  2m25s  nginx-ingress-controller  AppProtectPolicy default/dataguard-alarm was added or updated
+```
+Note that in the events section, we have a `Normal` event with the `AddedOrUpdated` reason, which informs us that the configuration was successfully applied.
+ 
+## Running App Protect in Debug Mode
+
+Running the Ingress Controller in debug mode will also run the App Protect module in debug mode. Follow the steps of [Running NGINX in the Debug Mode](/nginx-ingress-controller/troubleshooting/#running-nginx-in-the-debug-mode).
+
+## Known Issues
+
+When using the Ingress Controller with the App Protect module, the following issues might happen depending on the number of Ingress Resources with App Protect enabled in your cluster:
+
+### NGINX reload times increase
+
+The Ingress Controller reloads NGINX whenever there is a change that requires NGINX to reload to apply a new configuration. Without the App Protect module enabled, usual reload times are around 150ms. If App Protect module is enabled and is being used by any number of Ingress Resources, these reloads might take a few seconds instead. If you are running more than one instance of the Ingress Controller, because there is no order on how the Ingress Controller processes the Kubernete Reources, it might occur that different instances have different NGINX configurations until all of them have finished with the reloads.
+
+In order to reduce these inconsistencies, we recommend not to apply changes to resources handled by the Ingress Controller at the same time. 
+
+### NGINX fails to start or reload
+
+The first time the Ingress Controller starts or whenever there is a change that requires reloading NGINX, the Ingress Controller will verify if the reload was successful. There is a timeout for this verification. When App Protect is enabled, this timeout is 20 seconds (instead of 4 seconds without the module). 
+
+This timeout should be more than enough to verify configurations. However, when numerous Ingress Resources with App Protect enabled are handled by the Ingress Controller at the same time (for instance, if you apply a large amount of Ingress Resources at once, or when the Ingress Controller runs for the first time a cluster where the Ingress Resources with App Protect enabled are already present), this timeout might not be enough.
```

Comment:
```suggestion
This timeout should be more than enough to verify configurations. However, when numerous Ingress Resources with App Protect enabled are handled by the Ingress Controller at the same time, you may find that you need to extend the timeout further.  Examples of when this might be necessary include:

- You need to apply a large amount of Ingress Resources at once.
- You are running the Ingress Controller for the first time in a cluster where the Ingress Resources with App Protect enabled are already present.
```